<launch>

  <arg name="fixed_frame_id" default="map"/>
  <arg name="child_frame_id" default="velodyne"/>

  <include file="$(find velodyne_puck_driver)/launch/velodyne_puck_driver_nodelet.launch"/>

  <node pkg="nodelet" type="nodelet" name="velodyne_puck_decoder_nodelet"
    args="load velodyne_puck_decoder/VelodynePuckDecoderNodelet
    velodyne_puck_nodelet_manager"
    output="screen">
    <param name="fixed_frame_id" value="$(arg fixed_frame_id)"/>
    <param name="child_frame_id" value="$(arg child_frame_id)"/>

    <param name="min_range" value="0.3"/>
    <param name="max_range" value="100.0"/>
    <param name="frequency" value="2.0"/>
    <param name="publish_point_cloud" value="true"/>
  </node>

</launch>
